### PR TITLE
First prototype to display damage.

### PR DESCRIPTION
**Pull request by BigKahuna7385**

### CODE REVIEW

File: src/main/java/calculationEngine/entities/CeEntity.java
Comment (lkno0705) [inline src/main/java/calculationEngine/entities/CeEntity.java:209]: why are you creating a temporary list here which is just a copy from the already existing list?

File: src/main/java/calculationEngine/entities/CeEntity.java
Comment (lkno0705) [inline src/main/java/calculationEngine/entities/CeEntity.java:210]: Why do you clear the list here. The functionality of this method is a littlebit strange to me.

File: src/main/java/com/littleBeasts/GameLogic.java
Comment (lkno0705) [inline src/main/java/com/littleBeasts/GameLogic.java:34]: What is this List for? and why is it final? A final list can't be changed, which eliminates most of the UCs for a list.

File: src/main/java/com/littleBeasts/GameLogic.java
Comment (lkno0705) [inline src/main/java/com/littleBeasts/GameLogic.java:135]: If I understood the UC for this var correctly I would rather call it faceLeft or something like that. Left is a bit too general here I think

File: src/main/java/com/littleBeasts/entities/Beast.java
Comment (lkno0705) [inline src/main/java/com/littleBeasts/entities/Beast.java:25]: I would synchronise this with the CeEntity, where the var is called wild. Then we wouldn't need to invert the boolean. Minor change, not really necessary, but would give it better readability and beauty imo.

File: src/main/java/com/littleBeasts/entities/Player.java
Comment (lkno0705) [inline src/main/java/com/littleBeasts/entities/Player.java:42]: I guess this is also only for dev purposes?

File: src/main/java/com/littleBeasts/entities/Player.java
Comment (lkno0705) [inline src/main/java/com/littleBeasts/entities/Player.java:157]: For now ok. But if we add attacks we should change this. This should be Part of another Class e.g. PlayerAnimations, which will be more easy to maintain and will also improve readability.

File: src/main/java/com/littleBeasts/screens/BeastStats.java
Comment (lkno0705) [inline src/main/java/com/littleBeasts/screens/BeastStats.java:16]: See previous comment on this var name

File: src/main/java/com/littleBeasts/entities/Player.java
Comment (lkno0705) [inline src/main/java/com/littleBeasts/entities/Player.java:51]: I think beastToCeEntity isn't a good describing name for the method. Because it sounds like you're talking about 1 beast which gets converted.

File: src/main/java/com/littleBeasts/entities/Beast.java
Comment (lkno0705) [inline src/main/java/com/littleBeasts/entities/Beast.java:16]: I wouldn't call it littleBeast. Call it what it is a CeEntity or a ceEntityPointer / reference. Same applies for the get method

**Review by lkno0705 — CHANGES_REQUESTED** [review]
I won't approve it until further discussions about the comments I made. Also some things in this PR need to be described / explained to me before I can approve them.

File: src/main/java/calculationEngine/entities/CeAi.java
Comment (BigKahuna7385) [reply src/main/java/calculationEngine/entities/CeAi.java:46]: Constructor for only a single beast

File: src/main/java/calculationEngine/entities/CeEntity.java
Comment (BigKahuna7385) [reply src/main/java/calculationEngine/entities/CeEntity.java:209]: How would you do it?

File: src/main/java/calculationEngine/entities/CeEntity.java
Comment (BigKahuna7385) [reply src/main/java/calculationEngine/entities/CeEntity.java:210]: Otherwise it would just keep displaying all the damages that were dealt.

File: src/main/java/com/littleBeasts/GameLogic.java
Comment (BigKahuna7385) [reply src/main/java/com/littleBeasts/GameLogic.java:34]: To know which beasts are in battle atm

File: src/main/java/com/littleBeasts/entities/Beast.java
Comment (BigKahuna7385) [reply src/main/java/com/littleBeasts/entities/Beast.java:16]: You're right.

File: src/main/java/com/littleBeasts/entities/Player.java
Comment (BigKahuna7385) [reply src/main/java/com/littleBeasts/entities/Player.java:42]: No, it's the punch

File: src/main/java/com/littleBeasts/entities/Player.java
Comment (BigKahuna7385) [reply src/main/java/com/littleBeasts/entities/Player.java:51]: Lawand also said that. :-D I forgot to change it.

File: src/main/java/com/littleBeasts/entities/Player.java
Comment (BigKahuna7385) [reply src/main/java/com/littleBeasts/entities/Player.java:157]: Yeah, it will be moved to the Attack

File: src/main/java/com/littleBeasts/screens/BeastStats.java
Comment (BigKahuna7385) [reply src/main/java/com/littleBeasts/screens/BeastStats.java:16]: would "isPlayerBeast" be O.K.?

File: src/main/java/calculationEngine/entities/CeAi.java
Comment (BigKahuna7385) [reply src/main/java/calculationEngine/entities/CeAi.java:46]: For wild beasts

File: src/main/java/com/littleBeasts/GameLogic.java
Comment (BigKahuna7385) [reply src/main/java/com/littleBeasts/GameLogic.java:34]: the final was added by IntelliJ, I don't question the wisdom of IntelliJ. :-D

File: src/main/java/com/littleBeasts/entities/Player.java
Comment (BigKahuna7385) [reply src/main/java/com/littleBeasts/entities/Player.java:157]: I already talked with Lawand about that. Perhaps we can have different Kategories of Attacks, like Melee, Ranged and Spell, with their own animations.

File: src/main/java/com/littleBeasts/GameLogic.java
Comment (BigKahuna7385) [reply src/main/java/com/littleBeasts/GameLogic.java:135]: good Idea

File: src/main/java/com/littleBeasts/entities/Beast.java
Comment (BigKahuna7385) [reply src/main/java/com/littleBeasts/entities/Beast.java:25]: I'll look into it, sounds like a very good idea.

Comment (BigKahuna7385) [reply src/main/java/com/littleBeasts/entities/Beast.java:25]: Hmm.. just thought about it. "wild" has more implications than playerBeast. 
wild differenciates between a beast that can be caught or not. 
playerBeast is only for the GUI. So it is a tag, if the Beast should be rendered in the BattleHud or not.

File: src/main/java/calculationEngine/entities/CeEntity.java
Comment (BigKahuna7385) [reply src/main/java/calculationEngine/entities/CeEntity.java:209]: I want to return the list and then clear it. This was the way I found. :-D

**Review by lkno0705 — APPROVED** [review]

**Pull request merged**
